### ISSUE TITLE
Training Details

### ISSUE THREAD

**Issue by Junsheng121** [bytedance/InfiniteYou#39]
Hello! I'm currently working on reproducing the InfiniteYou model. I was wondering what learning rate scheduler was used in your training pipeline. Thanks a lot for your open-source contribution — it's greatly appreciated!

**Comment by EndlessSora** [comment]
Thanks a lot for your interest and effort in reproducing our training. We used Adam optimizer as mentioned in the paper. No specific scheduler was applied.
